instrumentation/jedis-4.0.0/src/main/java/redis/clients/jedis/JedisPubSub_Instrumentation.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *
+ *  * Copyright 2020 New Relic Corporation. All rights reserved.
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package redis.clients.jedis;
+
+import com.newrelic.agent.bridge.datastore.DatastoreVendor;
+import com.newrelic.api.agent.DatastoreParameters;
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.Trace;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.NewField;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+
+@SuppressWarnings({ "ResultOfMethodCallIgnored", "unused" })
+@Weave(type = MatchType.BaseClass, originalName = "redis.clients.jedis.JedisPubSub")
+public class JedisPubSub_Instrumentation {
+
+    @NewField
+    private String host;
+
+    @NewField
+    private int port;
+
+    @Trace
```

Comment:
Any reason to trace this new method?
It may cause confusion when people upgrade Jedis and see a new span in their traces.